orca-bakery/src/main/groovy/com/netflix/spinnaker/orca/bakery/tasks/CreateBakeTask.groovy

```diff
@@ -58,6 +59,9 @@ class CreateBakeTask implements RetryableTask {
   @Override
   TaskResult execute(Stage stage) {
     String region = stage.context.region
+    if (!bakery) {
+      throw new RuntimeException("You have not enabled baking for this orca instance. Set bakery.enabled = true")
```

Comment:
In most of the other messages you use this style: `bakery.enabled: true`. Here and one other spot you use `=`.